frontends/web/src/components/bluetooth/bluetooth.tsx

```diff
@@ -79,8 +107,25 @@ const _Bluetooth = () => {
         })}
       </div>
       {state.scanning && (
-        <HorizontallyCenteredSpinner />
+        <>
+          <HorizontallyCenteredSpinner />
+          {showConnectionIssues && (
+            <div className={styles.connectionIssuesLink}>
+              <ButtonLink
+                to="#"
+                transparent
+                onClick={(e) => {
+                  e.preventDefault();
+                  setDialogOpen(true);
+                }}
+              >
+                {t('novaConnectionIssues.havingConnectionIssues')}
```

Comment:
Prefix with `bluetooth.connectionIssues` would be a better namespace right? As it's in bluetooth.tsx, and it's not related to nova per se, but to bluetooth.